src/App.tsx

```diff
@@ -12,11 +20,15 @@ function App() {
     roles: ["director_producer"],
   };
   return (
-    <div className="App">
-      <header className="App-header">
-        <VideoCard {...video} />
-      </header>
-    </div>
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="contact" element={<Contact />} />
+        <Route path="videoListPage" element={<VideoListPage />} />
+        <Route path="series" element={<SeriesPage />} />
+        <Route path="animation" element={<AnimationPage />} />
```

Comment:
All of the examples I'm seeing on the [docs](https://reactrouter.com/en/main/route/route) use a `"/"` to start the routes. Do we need to do that?

```suggestion
        <Route path="/" element={<Home />} />
        <Route path="/contact" element={<Contact />} />
        <Route path="/videoListPage" element={<VideoListPage />} />
        <Route path="/series" element={<SeriesPage />} />
        <Route path="/animation" element={<AnimationPage />} />
```

Reply:
The routing was working fine so it doesn't seem necessary, but I  agree and will input the "/" to stick to the official docs. 
I was using this w3 article as an example while I wrote this and it does _not_ use the "/".
<img width="691" alt="Screen Shot 2024-01-08 at 8 30 57 AM" src="https://github.com/mwkwsd/sensensomething/assets/102488171/60776b92-3017-46ce-b969-7acd35e377c3">